cvnp/cvnp.cpp

```diff
@@ -127,6 +179,15 @@ namespace cvnp
         int type = detail::determine_cv_type(a, depth);
         cv::Size size = detail::determine_cv_size(a);
         cv::Mat m(size, type, is_not_empty ? a.mutable_data(0) : nullptr);
+
+        if (is_not_empty) {
+            m.u = detail::g_cvnpAllocator.allocate(a); //, ndims, size, type, step);
+            m.allocator = &detail::g_cvnpAllocator;
+
+            // this doesn't leak, but I don't know why
```

Comment:
I did check to see that the deallocator is eventually called. Things break if this addref isn't here.

Reply:
See note where I suggest to add a line `u->refcount = 1` inside allocate